install_config/configuring_local.adoc

```diff
@@ -208,3 +208,227 @@ $ chcon -R unconfined_u:object_r:svirt_sandbox_file_t:s0 /mnt/local-storage/
 ====
 Omitting any of these steps may result in the wrong PV being created.
 ====
+
+[[local-volume-raw-block-devices]]
+=== Raw Block Devices
+It is possible to statically provision also raw block devices using the local
+volume provisioner. This feature is disabled by default and requires additional
+configuration.
+
+. Enable the `BlockVolume` feature gate on all masters and nodes.
+Edit or create the master configuration file on all masters
+(*_/etc/origin/master/master-config.yaml_* by default) and add `BlockVolume=true`
+under the `apiServerArguments` and `controllerArguments` sections:
++
+[source, yaml]
+----
+apiServerArguments:
+   feature-gates:
+   - BlockVolumes=true
+...
+
+ controllerArguments:
+   feature-gates:
+   - BlockVolumes=true
+...
+----
+
+. On all nodes, edit or create the node configuration file
```

Comment:
For openshift 3.10, enabling feature gate of node service need update configmaps in "openshift-node" namespace and do not need restart node service manually.

Reply:
OK. Will fix this.